chapter_introduction/index_vn.md

```diff
@@ -833,7 +835,9 @@ and other times we might need to employ human annotators to label the data,
 (e.g., assigning images to categories).
 -->
 
-*dịch đoạn phía trên*
+Một cách dễ hiểu, quá trình học tương tự với:
+Lấy một tập mẫu lớn ở đó các hiệp biến đã biết trước và chọn từ chúng một tập con ngẫu nhiên, thu thập các nhãn chính xác cho chúng. <!-- cần thảo luận đưa ground truth vào glossary -->
```

Comment:
```suggestion
Lấy một tập mẫu lớn ở đó các hiệp biến đã biết trước.
Từ đó chọn ngẫu nhiên một tập con và  dựa trên cơ sở thực tế thu thập các nhãn tương ứng.
```

Reply:
"ground truth" không thể dịch là "cơ sở thực tế" được. Nghĩa của từ này là giá trị chính xác của nhãn. 

Mình chợ nhớ đến "bảng chân trị" (the truth table) được dùng trong các hàm logic (AND, NOT, OR, XOR). Phải chăng mình nên dùng từ "chân trị" để thay cho "ground truth"?

cc @ngcthuong @william-vu @nmdang.

Reply:
https://datascience.stackexchange.com/questions/17839/what-is-ground-truth
Thật ra "cơ sở thực tế" khá chính xác vì đó là giá trị nhận được trong quá trình thu thập và đo đạc dữ liệu. Em nghĩ từ "chân trị" trong vật lý điện tử nhằm nói đến "giá trị của chân" của con IC nên sẽ không dùng được trong bối cảnh này.

Tuy nhiên "cơ sở thực tế" hơi dài nên em cũng chưa thích lắm

Reply:
"chân trị" nghe khá hay, nhưng chính xác hay không thì em không có ý kiến. Chờ mọi người.

Reply:
@tiepvupsu : theo e hiểu thì "chân trị" là giá trị chân lý; 1 ví dụ: giá trị chân lý của một mệnh đề có thể đúng hoặc sai (e nhớ sách lớp 10 nói thế) --> có vẻ chưa phù hơp khi dịch "ground truth" thành "chân trị".

Reply:
> Update: sau khi nghĩ kĩ thì e cũng ko biết là đúng hay sai. Có thể dịch là "giá trị thật của nhãn" ko bác?

Theo một số tài liệu thì nó chính là "nhãn" chứ không cần phải ghi "thật của nhãn" nữa.
Có phần này quan trọng có lẽ mình cũng lưu ý khi dịch:
> Ground truth can be wrong. It is a measurement, and there can be errors in it. In some ML scenarios it can also be a subjective measurement where it is difficult define an underlying objective truth - e.g. expert opinion or analysis, which you are hoping to automate. Any ML model you train will be limited by the quality of the ground truth used to train and test it, and that is part of the explanation on the Wikipedia quote. It is also why published articles about ML should include full descriptions of how the data was collected.

Nguồn: https://datascience.stackexchange.com/a/17840

Reply:
Mình thấy wikipedia có viết thế này:

> Ground truth is a term used in various fields to refer to information provided by direct observation (i.e. empirical evidence) as opposed to information provided by inference.

Nếu muốn nhấn mạnh vào ý "empirical evidence" thì mình cũng có thể dịch ground truth (tính từ) thành "quan trắc". Tỷ dụ ground truth data là dữ liệu quan trắc, ground truth label --> nhãn quan trắc

Reply:
Mình cũng thấy từ "quan trắc" khá gần rồi. Tuy nhiên, mình thấy từ này có vẻ chỉ được dùng trong trường hợp nói về môi trường hay khí hâu [như ở đây](http://tratu.soha.vn/dict/vn_vn/Quan_tr%E1%BA%AFc).

Mình chờ thêm ý kiến.

Reply:
hmm. Ground truth như mình dùng bên mình thường theo nghĩa là "gốc". Ảnh gốc, ảnh chưa qua chỉnh sửa chưa bị suy hao các thứ. 

Còn các từ ở trên thì mình thấy "trân trị", thì tớ cùng quan điểm với @william-vu. Quan trắc thì giống từ liên quan tới đo đạc hơn, cảm giác không khái quát cho tất cả các trường hợp được. 
Đặt thử các từ khác như Ground truth image ---> ảnh quan trắc thì nghe nó ra từ khác rồi.

Reply:
PR này đang bị blocked bởi #185

Reply:
Mình thấy "nhãn gốc" ok, từ "gốc" có thể coi như gần với "ground". Có lẽ mình sẽ dịch là "nhãn gốc" rồi khi có thay đổi sẽ đổi lại sau.

Reply:
@lkhphuc ^

Reply:
Em đang thắc mắc là `ground truth label` dịch là `nhãn gốc` thì OK, `ground truth bounding box` hay `ground truth + Noun` thì dịch là sao cho phù hợp? ví dụ: `khung chứa gốc`..

Reply:
@davidnvq anh mở thread để thảo luận giúp em nhé

Reply:
mở PR mới ấy à @rootonchair?

Reply:
Vâng ạ anh @davidnvq